spec/index.bs

```diff
@@ -1830,6 +1830,9 @@ dictionary GPUBindGroupLayoutEntry {
     // Used for uniform buffer and storage buffer bindings.
     boolean hasDynamicOffset = false;
 
+    // Used for uniform buffer and storage buffer bindings.
+    GPUSize64 minimumBufferBindingSize = 0;
```

Comment:
That's a mouthful, why not `minBufferSize` or `minimumBufferSize`?

Reply:
Could be `minBufferBindingSize` for sure.
We can't throw out the "binding" in here though, since we aren't talking about the whole buffer size, and that should be clear from the name.